### PR TITLE
feat: landing revamp

### PR DESCRIPTION
**Pull request by Chef-Yogi**
### Summary
🎨🚀🌎

This pull request redesigns the home page of the PancakeSwap web app to highlight its community, ecosystem, and multichain features. It adds new components and sections such as `CakeSection`, `CommunitySection`, `EcoSystemSection`, `MetricsSection`, and `NewsSection`, and removes or renames old components and sections such as `WinSection` and `Hero`. It also updates the styles, images, and texts of the home page to match the new design and branding.

> _Sing, O Muse, of the glorious deeds of PancakeSwap, the multichain DEX_
> _That offers many features and products to its users, both old and new_
> _And how its skilled developers, with wisdom and craft, updated its home page_
> _To display its performance, community, and ecosystem, shining and true_

### Walkthrough
*  Add a new component `CakeSection` to display information and tags about the CAKE token and its ecosystem ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-2f5688e75a164e9b748ebdee24ceae9b7562a789fd64db4b15be2b7c1854538bR1-R141), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-7586b0130adcd1857a6aa012be90ac79f2345e939dcf376fbe72b2af655e362dR1-R47))
*  Rename the `WinSection` component to `CommunitySection` and update its content and style to showcase the PancakeSwap community and its social media platforms ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-ac34be30516adf91021a95ec5131330847cab7148addaa3e66c462a3b60870b0L1-R8), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-ac34be30516adf91021a95ec5131330847cab7148addaa3e66c462a3b60870b0L57-L70), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-ac34be30516adf91021a95ec5131330847cab7148addaa3e66c462a3b60870b0L92-R77), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-ac34be30516adf91021a95ec5131330847cab7148addaa3e66c462a3b60870b0L108-R102), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-ac34be30516adf91021a95ec5131330847cab7148addaa3e66c462a3b60870b0L120-R116), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-ac34be30516adf91021a95ec5131330847cab7148addaa3e66c462a3b60870b0L136-R122), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-4ff12c73ec3223f1e54309aea2cbbb74a8749128757eb7fa7fab956fe80f3cb5), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-413567f1660e2ebe00f10e81aa88b7a26f80cf7f8d4aecd0b38efba588992ca2))
*  Add new components `CommunitySummary` and `CommunityTags` to render statistics and icons related to the community ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-4c1f2773bbb3aa1cc623ac49724324dcd5bef5b10051a786b8b152eb0ba61a62R1-R53), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-7255e2de038584eaae9ff1c5d590fc0706ac021bb6fae1d83c50f7ef96cfecb5R1-R61))
*  Add a new component `TwitterCards` to render the top tweet of the week from the PancakeSwap account ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-7896a3ab50523c8e73d09ace768c9c5bc9f813d66bef4883686dd112e603b12fR1-R102))
*  Add a new component `EcoSystemSection` to display cards featuring the different features and products that PancakeSwap offers in its ecosystem ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-eb773e70dc574977f184dfbdfca993e7fe13a5995b9f433ab192864c6ee2c6e5R1-R166))
*  Add a new component `MetricsSection` to display statistics and metrics about PancakeSwap's performance and growth ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-5bbdfe74df9f994dd43a2f88a8386381f05462c0f7dafd39d207d5c669055e77R1-R43))
*  Add a new component `ChainTags` to render tags representing the different chains that PancakeSwap supports or plans to support ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-4bd58b43c2b32ab0e9c9920537b01ce303c496887ea6fdfd56cc43f6d012c7b9R1-R49))
*  Add a new component `NewsSection` to display cards featuring the latest news and articles about PancakeSwap from various sources and platforms ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-cda12b23cd0cd526f330ba3a431e557d9eaef2fef93065496a35af6aa2864ff2L1-R70))
*  Update the `Hero` component to use a new background image and heading text that reflect the main value proposition of PancakeSwap as an all-in-one multichain DEX ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-f627b9b93a193fa7b84d2cd90920cd603842811dc2d95eda901327f3c03b0f39R12), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-f627b9b93a193fa7b84d2cd90920cd603842811dc2d95eda901327f3c03b0f39L129-R135), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-f627b9b93a193fa7b84d2cd90920cd603842811dc2d95eda901327f3c03b0f39L152-R158))
*  Update the `Footer` component to use a new heading text that reflects the main value proposition of PancakeSwap as an all-in-one multichain DEX and remove unnecessary text and link ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-b2058c2333fb75776e84abff92fe58417b5adc3850a5d5f46f71ff7dee4fbc09L106-R108))
*  Update the `Home` view to render the new components and sections in the appropriate order and with the appropriate styles and props, and remove or comment out the old components and sections that are no longer relevant or used ([link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-4b52c64bc7f1cb355abbf1d2fbb926eb075b51bdbfecca67208f26fc84060f82R1-R138), [link](https://github.com/pancakeswap/pancake-frontend/pull/7398/files?diff=unified&w=0#diff-a3036a0bb4c8967f7a13f579b01902e65cde700b6268c495ca2dc48a37e3ebc9L1-R27))

### CODE REVIEW

File: apps/web/src/views/Home/components/CommunitySection/TwitterCards.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+export const useLatestBlogData = () => {
+  return useMemo(() => {
+    return {
+      title: 'Step by Step Guide to Use Arbitrum PancakeSwap v3',
+      imageSrc: 'https://sgp1.digitaloceanspaces.com/strapi.space/b477fb73428ef9accc2b312873e9140c.jpg',
+      link: 'https://blog.pancakeswap.finance/articles/step-by-step-guide-to-use-arbitrum-one-pancake-swap-v3',
+      date: 'Aug 10, 2023',
+    }
+  }, [])
+}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CommunitySection/TwitterCards.tsx:77]: Static data?

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 2]
```diff
@@ -3,24 +3,27 @@ import { FooterLinkType } from "../../../components/Footer/types";
 
 export const footerLinks: (t: ContextApi["t"]) => FooterLinkType[] = (t) => [
   {
-    label: t("About"),
+    label: "Ecosystem",
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Menu/components/footerConfig.ts:6]: miss `t`

**Comment by chef-jojo** [comment]
That's a lot of assets, can we move it to cdn?

File: packages/localization/src/config/translations.json [hunk 3]
```diff
@@ -2700,5 +2678,55 @@
...
+  "Featured": "Featured",
+  "News": "News",
+  "Figures": "Figures",
+  "Tokenomics": "Tokenomics",
+  "Farms and Syrup Pools": "Farms and Syrup Pools",
+  "Multichain Deployment": "Multichain Deployment",
+  "Etherum Expansion": "Etherum Expansion",
+  "Aptos Development": "Aptos Development",
+  "Contributing": "Contributing",
+  "Github": "Github",
```
Comment (chef-jojo) [inline packages/localization/src/config/translations.json:2729]: No need to translate

File: packages/localization/src/config/translations.json [hunk 4]
```diff
@@ -2700,5 +2678,55 @@
...
+  "Immersive PvP & PvE tower-defense GameFi": "Immersive PvP & PvE tower-defense GameFi",
+  "Enter for a chance to win CAKE prize pools": "Enter for a chance to win CAKE prize pools",
+  "Stake CAKE acquire pottery tickets win prizes": "Stake CAKE acquire pottery tickets win prizes",
+  "Trade unique NFTs on BNB Chain": "Trade unique NFTs on BNB Chain",
+  "Discover the": "Discover the",
+  "Game & NFT": "Game & NFT",
+  "Join Everyone's Favorite DEX Now!": "Join Everyone's Favorite DEX Now!",
+  "Everyone's": "Everyone's",
+  "Favorite": "Favorite",
+  "DEX": "DEX",
```
Comment (chef-jojo) [inline packages/localization/src/config/translations.json:2713]: Umm it's questionable should it be translate text

File: packages/uikit/src/components/Svg/Icons/ZksyncWithOutCircle.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Svg from "../Svg";
+import { SvgProps } from "../types";
+
+export const ZksyncWithOutCircleIcon: React.FC<React.PropsWithChildren<SvgProps>> = (props) => {
+  return (
+    <Svg viewBox="0 0 27 15" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```
Comment (chef-jojo) [inline packages/uikit/src/components/Svg/Icons/ZksyncWithOutCircle.tsx:7]: For some svgs that no need to manipulate, can consider just load svg by img?

File: apps/web/src/views/Home/hooks/useDrawCanvas.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    context.clearRect(0, 0, width, height)
+    context.drawImage(video, 0, 0, width, height)
+  }, [canvas, video, height, width])
+
+  if (isElementReady) {
+    video.onpause = () => {
+      clearInterval(canvasInterval)
+    }
+
+    video.onended = () => {
```
Comment (chef-jojo) [inline apps/web/src/views/Home/hooks/useDrawCanvas.ts:29]: event listener should be inside useEffect

File: packages/uikit/src/components/Svg/Icons/ZksyncWithOutCircle.tsx (hunk 5 above)
Comment (Chef-Yogi) [reply packages/uikit/src/components/Svg/Icons/ZksyncWithOutCircle.tsx:7]: ok, sure

File: apps/web/src/views/Home/hooks/useDrawCanvas.ts (hunk 6 above)
Comment (Chef-Yogi) [reply apps/web/src/views/Home/hooks/useDrawCanvas.ts]: fixed

File: apps/web/src/views/Home/components/Banners/MultipleBanner.tsx [hunk 7]
```diff
@@ -115,7 +112,7 @@ const MultipleBanner: React.FC<React.PropsWithChildren> = () => {
   }, [bannerList, swiperRef])
 
   return (
-    <BannerPlaceHolder walletConnected={Boolean(account) && chainId === ChainId.BSC}>
+    <BannerPlaceHolder walletConnected={false}>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/MultipleBanner.tsx:115]: If `walletConnected ` always false. Can u clear the walletConnected  code in css?

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 8]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
-        <Text color="textSubtle">{t('Burned to date')}</Text>
+      <StyledColumn>
+        <Text bold>{t('Token Burn')}</Text>
         {burnedBalance ? (
-          <Balance decimals={0} lineHeight="1.1" fontSize="24px" bold value={burnedBalance} />
+          <Balance color="secondary" decimals={0} lineHeight="1.1" fontSize="24px" bold value={burnedBalance} />
         ) : (
           <Skeleton height={24} width={126} my="4px" />
         )}
+        {/* <Text color="textSubtle">{t('Unit')}</Text> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:174]: Clean up all unused code

File: apps/web/src/views/Home/components/CommunitySection/CommunitySummary.tsx [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+        </Text>
+        <StyledText fontSize="40px" bold mb="27px">
+          15 +
+        </StyledText>
+        <Text color="white" bold fontSize="16px">
+          {t('Community Ambassadors')}
+        </Text>
+        <StyledText fontSize="40px" bold mb="27px">
+          35 +
+        </StyledText>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CommunitySection/CommunitySummary.tsx:61]: Static data?

Comment (Chef-Yogi) [reply apps/web/src/views/Home/components/CommunitySection/CommunitySummary.tsx:61]: yes

File: apps/web/src/views/Home/components/CommunitySection/TwitterCards.tsx (hunk 1 above)
Comment (ChefMomota) [reply apps/web/src/views/Home/components/CommunitySection/TwitterCards.tsx:94]: Use blog API get the latest data

File: apps/web/src/views/Home/components/CommunitySection/TwitterCards.tsx [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+
+export const BlogCard: React.FC = () => {
+  const { t } = useTranslation()
+  const { imageSrc, title, date, link } = useLatestBlogData()
+  const { theme } = useTheme()
+  return (
+    <Wrapper
+      onClick={() => {
+        window.open(link, '_blank', 'noopener noreferrer')
+      }}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CommunitySection/TwitterCards.tsx:159]: Change it to function name

File: apps/web/src/views/Home/components/CommunitySection/index.tsx [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+// const topRightImage = {
+//   path: '/images/home/community-items/',
+//   attributes: [
+//     { src: '5', alt: 'Lottery ball number 9' },
+//     { src: '1', alt: 'Lottery ball number 2' },
+//     { src: '2', alt: 'Lottery ball number 4' },
+//     { src: '3', alt: 'Lottery ball number 6' },
+//     { src: '4', alt: 'Lottery ball number 7' },
+//   ],
+// }
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CommunitySection/index.tsx:110]: Remove

File: apps/web/src/views/Home/components/CommunitySection/index.tsx [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+
+  return (
+    <>
+      <BgWrapper>
+        {/* <BottomLeftImgWrapper>
+          <CompositeImage {...bottomLeftImage} />
+        </BottomLeftImgWrapper>
+        <TopRightImgWrapper>
+          <CompositeImage {...topRightImage} />
+        </TopRightImgWrapper> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CommunitySection/index.tsx:125]: Remove

File: apps/web/src/views/Home/components/MetricsSection/index.tsx [hunk 13]
```diff
@@ -1,85 +1,124 @@
...
-      <Heading textAlign="center" scale="xl">
-        {t('Used by millions.')}
-      </Heading>
-      <Heading textAlign="center" scale="xl" mb="32px">
-        {t('Trusted with billions.')}
-      </Heading>
-      <Text textAlign="center" color="textSubtle">
-        {t('PancakeSwap has the most users of any decentralized platform, ever.')}
+    <Flex justifyContent="center" alignItems="center" flexDirection="column" overflow="hidden">
+      {/* <GradientLogo height="48px" width="48px" mb="24px" /> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/MetricsSection/index.tsx:62]: Remove

File: apps/web/src/views/Home/hooks/getArticle.ts [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+    return (response.data as ResponseCategoriesType[]).map((category) => ({
+      id: category.id,
+      name: category.attributes.name,
+    }))
+  } catch (error) {
+    console.error('[ERROR] Fetching Categories', error)
+    return []
+  }
+}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/hooks/getArticle.ts:74]: If not use it,  do not copy the code

File: apps/web/src/views/Home/index.tsx [hunk 15]
```diff
@@ -137,9 +122,47 @@ const Home: React.FC<React.PropsWithChildren> = () => {
...
+        <SalesSection {...swapSectionData(t)} />
+      </PageSection> */}
+      {/* <PageSection
+        innerProps={{ style: HomeSectionContainerStyles }}
+        background={theme.colors.gradientCardHeader}
+        index={2}
+        hasCurvedDivider={false}
+      >
         <SalesSection {...earnSectionData(t)} />
         <FarmsPoolsRow />
```
Comment (ChefMomota) [inline apps/web/src/views/Home/index.tsx:164]: Remove

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 16]
```diff
@@ -3,86 +3,132 @@ import { FooterLinkType } from "../../../components/Footer/types";
...
-        href: "https://pancakeswap.finance/terms-of-service",
-      },
-      {
-        label: "—",
-      },
       {
         label: t("Merchandise"),
         href: "https://merch.pancakeswap.finance/",
-        isHighlighted: true,
+        // isHighlighted: true,
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Menu/components/footerConfig.ts:39]: Confirm with Oreo, should we remove this

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 17]
```diff
@@ -3,86 +3,132 @@ import { FooterLinkType } from "../../../components/Footer/types";
...
+      //   href: "https://docs.pancakeswap.finance/multichain",
+      // },
+      // {
+      //   label: t("Etherum Expansion"),
+      //   href: "/swap",
+      // },
+      // {
+      //   label: t("Aptos Development"),
+      //   href: "/swap",
+      // },
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Menu/components/footerConfig.ts:69]: ?

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 18]
```diff
@@ -3,86 +3,132 @@ import { FooterLinkType } from "../../../components/Footer/types";
...
         href: "https://github.com/pancakeswap",
       },
+      {
+        label: t("Bug Bounty"),
+        href: "https://docs.pancakeswap.finance/developers/bug-bounty",
+      },
+      // {
+      //   label: t("Audits"),
+      //   href: "https://docs.pancakeswap.finance/help/faq#is-pancakeswap-safe-has-pancakeswap-been-audited",
+      // },
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Menu/components/footerConfig.ts:90]: Remove

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 19]
```diff
@@ -3,86 +3,132 @@ import { FooterLinkType } from "../../../components/Footer/types";
...
+    label: t("Support"),
+    items: [
+      {
+        label: t("Contact"),
+        href: "https://docs.pancakeswap.finance/contact-us/customer-support",
+      },
+      // {
+      //   label: t("Customer Support"),
+      //   href: "https://docs.pancakeswap.finance/contact-us/customer-support",
+      // },
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Menu/components/footerConfig.ts:103]: Remove

File: apps/web/src/components/Menu/config/config.ts [hunk 20]
```diff
@@ -116,24 +121,27 @@ const config: (
...
+    //   supportChainIds: SUPPORT_ONLY_BSC,
+    //   items: [
+    //     {
+    //       label: t('Trading Competition'),
+    //       href: '/competition',
+    //       image: '/images/decorations/tc.png',
+    //       hideSubNav: true,
+    //     },
+    //   ],
+    // },
```
Comment (ChefMomota) [inline apps/web/src/components/Menu/config/config.ts:138]: Remove

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 21]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
         {circulatingSupply ? (
-          <Balance decimals={0} lineHeight="1.1" fontSize="24px" bold value={circulatingSupply} />
+          <Balance decimals={0} lineHeight="1.1" fontSize="24px" bold value={circulatingSupply} color="secondary" />
         ) : (
           <Skeleton height={24} width={126} my="4px" />
         )}
-      </Flex>
-      <StyledColumn noMobileBorder style={{ gridArea: 'b' }}>
-        <Text color="textSubtle">{t('Total supply')}</Text>
+        {/* <Text color="textSubtle">{t('Unit')}</Text> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:137]: Remove

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 22]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
         {cakeSupply ? (
-          <Balance decimals={0} lineHeight="1.1" fontSize="24px" bold value={cakeSupply} />
+          <Balance color="secondary" decimals={0} lineHeight="1.1" fontSize="24px" bold value={cakeSupply} />
         ) : (
           <>
             <div ref={observerRef} />
             <Skeleton height={24} width={126} my="4px" />
           </>
         )}
+        {/* <Text color="textSubtle">{t('Unit')}</Text> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:149]: Remove

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 23]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
       </StyledColumn>
-      <StyledColumn noMobileBorder style={{ gridArea: 'c' }}>
+      {/* <StyledColumn noMobileBorder style={{ gridArea: 'c' }}>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:151]: Remove

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 24]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
         <Text color="textSubtle">{t('Max Supply')}</Text>
 
         <Balance decimals={0} lineHeight="1.1" fontSize="24px" bold value={750000000} />
-      </StyledColumn>
-      <StyledColumn noDesktopBorder style={{ gridArea: 'd' }}>
-        <Text color="textSubtle">{t('Market cap')}</Text>
+      </StyledColumn> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:155]: Remove

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 25]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
+        <Text bold>{t('Market cap')}</Text>
         {mcap?.gt(0) && mcapString ? (
-          <Heading scale="lg">{t('$%marketCap%', { marketCap: mcapString })}</Heading>
+          <Heading color="secondary" scale="lg">
+            {t('$%marketCap%', { marketCap: mcapString })}
+          </Heading>
         ) : (
           <Skeleton height={24} width={126} my="4px" />
         )}
+        {/* <Text color="textSubtle">{t('Unit')}</Text> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:165]: Remove

File: apps/web/src/views/Home/components/CakeDataRow.tsx [hunk 26]
```diff
@@ -130,57 +124,68 @@ const CakeDataRow = () => {
...
 
         {emissionsPerBlock ? (
-          <Heading scale="lg">{t('%cakeEmissions%/block', { cakeEmissions: formatNumber(emissionsPerBlock) })}</Heading>
+          <Heading color="secondary" scale="lg">
+            {t('%cakeEmissions%/block', { cakeEmissions: formatNumber(emissionsPerBlock) })}
+          </Heading>
         ) : (
           <Skeleton height={24} width={126} my="4px" />
         )}
+        {/* <Text color="textSubtle">{t('Unit')}</Text> */}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeDataRow.tsx:186]: Remove

File: apps/web/src/views/Home/components/CakeSection/CakeSectionTag.tsx [hunk 27]
```diff
@@ -0,0 +1,219 @@
...
+      },
+      {
+        icon: <TrophyIcon />,
+        text: t('Game'),
+      },
+      { icon: <GovernanceIcon />, text: t('Governance') },
+      { icon: <StoreIcon />, text: t('IFO') },
+    ]
+  }, [t])
+}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/CakeSection/CakeSectionTag.tsx:55]: Can we make all of this be like

const partnerData = [
 { icon: <PartnerLedger />, width: 21, text: <Trans>Ledger</Trans> },
]

File: apps/web/src/views/Home/components/CommunitySection/CommunityTags.tsx [hunk 28]
```diff
@@ -0,0 +1,63 @@
...
+    return [
+      { icon: <TwitterIcon />, href: 'https://twitter.com/pancakeswap', alt: 'TwitterIcon' },
+      { icon: <TelegramIcon />, href: 'https://t.me/pancakeswap', alt: 'TelegramIcon' },
+      { icon: <DiscordIcon />, href: 'https://discord.gg/pancakeswap', alt: 'DiscordIcon' },
+      { icon: <InstagramIcon />, href: 'https://www.instagram.com/pancakeswap_official/', alt: 'InstagramIcon' },
+      { icon: <RedditIcon />, href: 'https://www.reddit.com/r/pancakeswap/', alt: 'RedditIcon' },
+      { icon: <YoutubeIcon />, href: 'https://www.youtube.com/@pancakeswap_official', alt: 'YoutubeIcon' },
+    ]
+  }, [])
+}
```
[the remark this answers is not in the document]
Comment (ChefMomota) [reply apps/web/src/views/Home/components/CommunitySection/CommunityTags.tsx:54]: ?

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 29]
```diff
@@ -30,34 +33,38 @@ export const footerLinks: (t: ContextApi["t"]) => FooterLinkType[] = (t) => [
...
       },
-      {
-        label: t("Terms Of Service"),
-        href: "https://pancakeswap.finance/terms-of-service",
-      },
     ],
   },
   {
-    label: t("Help"),
+    label: "Business",
```
[the remark this answers is not in the document]
Comment (ChefMomota) [reply packages/uikit/src/widgets/Menu/components/footerConfig.ts:44]: update here

File: apps/web/src/views/Home/components/NewsSection/index.tsx [hunk 30]
```diff
@@ -0,0 +1,151 @@
...
+  {
+    platform: 'cointelegraph',
+    title: 'PancakeSwap wants to cap token inflation rate at 3%–5% per year',
+    description:
+      'Decentralized exchange (DEX) PancakeSwap wants to lower its token inflation to anywhere between 3% and 5% per annum, far below current rates of over 20%.',
+    imageSrc:
+      'https://images.cointelegraph.com/images/1434_aHR0cHM6Ly9zMy5jb2ludGVsZWdyYXBoLmNvbS91cGxvYWRzLzIwMjMtMDQvOWYxOGU4YjMtM2M0Ni00YjkxLTkzMDktM2RmNGU0ZGZjOWIwLmpwZw==.jpg',
+    link: 'https://cointelegraph.com/news/pancakeswap-wants-to-cap-token-inflation-rate-between-3-to-5-per-annum',
+    date: '2023-04-18',
+  },
```
[the remark this answers is not in the document]
Comment (Chef-Yogi) [reply apps/web/src/views/Home/components/NewsSection/index.tsx]: yes

**Pull request closed without merging**